.github/workflows/build.yml

```diff
@@ -32,8 +32,27 @@ jobs:
       matrix:
         python-version: [3.7]
         backend: ['django']
+    
+    services:
+      postgres:
+        image: postgres:12
+        env:
```

Comment:
This is very cool! I didn't know Github Actions could do this .

Reply:
Yeah. In previous versions, we had to install within the workflow. Now, it is already in container :)